packages/node_modules/@webex/webex-core/src/lib/credentials/token.js

```diff
@@ -377,11 +377,6 @@ const Token = WebexPlugin.extend({
         // found a bug in CI; log the details and reject the Promise
         if (this.access_token === obj.access_token) {
           this.logger.error('token: new token matches current token');
-          // log the tokens if it is not production
-          if (process.env.NODE_ENV !== 'production') {
```

Comment:
Same

Reply:
why are we removing these logs again? this should not be logged out in prod. was not the problem only concerning in prod mode? that's what I got from the JIRA.

Reply:
@akoushke - already had a discussion with Andrew and other members and come to conclusion that we should never log PII. Please let me know if its still a concern

Reply:
@AMANDUA while we appreciate @andrewholsted's feedback, the SDK core team needs to be aware of the changes as well. This can be done via the JS-SDK/Widgets Contributors Engineering space or directly with the product owner Olivier Profitt.

Reply:
okay @adamweeks what ever suits you. Let me know.

Reply:
> already had a discussion with Andrew , Chris and other members and come to conclusion that we should never log PII. Please let me know if its still a concern

This was not my intent. I am ok logging in dev but it should never be sent to a third party service.